.github/workflows/release.yaml

```diff
@@ -43,3 +43,15 @@ jobs:
           platforms: linux/amd64
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+
+  helm_release:
```

Comment:
This release isn't copying the newest CRD to the Helm charts repo (which has a duplicate CRD definition)

Reply:
It is done when the `./.github/workflows/helm_release.yaml` workflow invokes `make prepare-helm-release`.
I've removed the CRD content from the helm chart to avoid confusion.